traffic_portal/Dockerfile

```diff
@@ -14,7 +14,7 @@
 FROM node:4-onbuild
 #FROM buildpack-deps:jessie
 
-RUN apt-get update -y && apt-get install libffi-dev ruby-dev rubygems vim -y
+RUN apt-get update -y && apt-get --no-install-recommends install -y libffi-dev ruby-dev rubygems vim -y
```

Comment:
See above—I would prefer disabling `APT::Install-Recommends` and `APT::Install-Suggests` in the APT configuration to passing `--no-install-recommends` for every apt or apt-get command.